models/Recipe.js

```diff
@@ -0,0 +1,14 @@
+import mongoose from "mongoose";
+const { Schema } = mongoose;
```

Comment:
Just for the consistency of your notation: Wouldn't this variable have to be defined after the imports?